src/app/textarea/textarea.demo.html

```diff
@@ -43,6 +43,16 @@
           <textarea soho-textarea name="required" [(ngModel)]="model.requiredText" data-validate="required"></textarea>
         </div>
 
+        <div class="field">
+          <label for="growInfinite" class="label">Grow (Infinite)</label>
+          <textarea soho-textarea name="growInfinite" data-options="{'autoGrow': true}" [(ngModel)]="model.growableText"></textarea>
+        </div>
+
+        <div class="field">
+          <label for="growFinite" class="label">Grow (Finite)</label>
+          <textarea soho-textarea name="growFinite" data-options="{'autoGrow': true, 'autoGrowMaxHeight': 500}" [(ngModel)]="model.growableText"></textarea>
+        </div>
+
```

Comment:
data-options is one way ids-enterprise reads in option values.

To use the new @inputs they should be use 1 of 2 ways:
1. as initializers
```
<textarea soho-textarea name="growFinite" 
   autoGrow="true"
   autoGrowMaxHeight="'500'"
   [(ngModel)]="model.growableText"
></textarea>
```
2. as bound variables/values
```
<textarea soho-textarea name="growFinite" 
   [autoGrow]="this.autoGrowTextArea"        <-- where autoGrowTextArea is a variable in your component
   [autoGrowMaxHeight]="this.autoGrowToMaxHeight"  <-- where autoGrowToMaxHeight is a variable in your component
   [(ngModel)]="model.growableText"
></textarea>
```

Reply:
If `autoGrowMaxHeight` is a number you don't need the quotes?